app/elements/cardlist-api/card-list.html

```diff
@@ -20,18 +26,34 @@
 
       properties: {
         collection: {
-          type: Array
+          type: Array,
+          value: [],
+          observer: 'reFilter'
         },
         name: {
           type: String
         },
         user: {
           type: Object
+        },
+        selected: {
+          type: Number,
+          value: 0,
+          observer: 'reFilter'
+        },
+        groups: {
+          type: Array
+        },
+        items: {
+          type: Array
         }
       },
 
-      updateCollection: function(event) {
-        this.splice('collection', event.detail.index, 1);
+      removeDroppedItem: function(event) {
+        let droppedElement = this.items[event.detail.index];
```

Comment:
Oops, we should not use let, since some browsers do not support it yet **badum tss**